tests/test_chi_ssa_69.py

```diff
@@ -0,0 +1,132 @@
+from os.path import dirname, join
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_69 import ChiSsa69Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_69.json"),
+    url="https://auburngresham.wixsite.com/ssa69/calendar",
+)
+
+spider = ChiSsa69Spider()
+
+freezer = freeze_time("2019-10-09")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+def test_title():
+    assert parsed_items[0]["title"] == 'SSA #69 Commissioners'
+    assert parsed_items[1]["title"] == 'CPD 006th District C.A.P.S Business Subcommittee Meetings'
+    assert parsed_items[2]["title"] == 'CPD 006th District C.A.P.S Business Subcommittee Meetings'
+
+
+def test_description():
+    assert parsed_items[0]["description"] == (
+        ' SSA #69 Commissioners Meeting:\n'
+        ' Corresponding Dates: \n'
+        'February 12, 2019; July 23, 2019; and  November 12, 2019\n'
+        ' Location: 7901 S. Racine, Chicago, IL 60620\n'
+        ' Time: 8:30 am to 10:00 am'
+    )
+    assert parsed_items[1]["description"] == (
+        'CPD 006th District C.A.P.S, Business Subcommittee Meetings\n'
+        'Dates: 2nd Tuesday of each month\n'
+        'Location: 006th District Police Station Community Room\n'
+        '7808 S. Halsted , Chicago, IL 60620'
+    )
+    assert parsed_items[2]["description"] == (
+        'CPD 006th District C.A.P.S, Business Subcommittee Meetings\n'
+        'Dates: 2nd Tuesday of each month\n'
+        'Location: 006th District Police Station Community Room\n'
+        '7808 S. Halsted , Chicago, IL 60620'
+    )
+
+
+def test_start():
+    assert str(parsed_items[0]["start"]) == "2019-11-12 09:30:00"
+    assert str(parsed_items[1]["start"]) == "2019-10-08 18:00:00"
+    # I think this recurring meeting has different start times
```

Comment:
These comments can be removed once the other issue is addressed

Reply:
done